acceptance-tests/src/test/resources/YubiShellSimulator

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/java --source 11
+
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import java.io.Console;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+// a dumb YubiHSM simulator for AT which prints BLS or SECP private key on standard output
+public class YubiShellSimulator {
+    private static final Map<String, String> argsMap = new HashMap<>();
+
+    public static void main(String[] args) {
+        for(String arg: args) {
+            final String[] kv = arg.split("=");
+            if (kv == null || kv.length != 2) {
+                System.err.println("Argument should be in format --arg=value");
+                System.exit(-1);
+            }
+            argsMap.put(kv[0], kv[1]);
+        }
+
+        // make sure required arguments are passed
+        if (!argsMap.containsKey("--connector")) {
+            System.err.println("--connctor is missing");
+            System.exit(-1);
+        }
+        if (!argsMap.containsKey("--authkey")) {
+            System.err.println("--authkey is missing");
+            System.exit(-1);
+        }
+        if (!argsMap.containsKey("--object-id")) {
+            System.err.println("--bject-id is missing");
```

Comment:
typo: should be "--object-id is missing"

Reply:
fixed.